### DISCUSSION TITLE
[help wanted] How to get the plugin's options in clientAppEnhanceFiles？

### DISCUSSION THREAD

**Discussion by recoluan** [vuepress/core#98] [Q&A] [answered]
I want to do something in ClientAppendanceFiles that relies on the plugin's options, but now that the Node API and the Client API are separate from each other. So what should I do?

**Comment by Mister-Hope** [comment, marked as answer]
use `define`.

Here is a demo(in V1, should be similar in V2): 

https://github.com/vuepress-theme-hope/vuepress-theme-hope/blob/9c7b58a3e24cced5d623da0bb1a53e80f1219e87/packages/comment/src/node/index.ts#L23-L28

**Reply by recoluan** [reply]
Thanks. I get it.
